.gitignore

```diff
@@ -1,2 +1,6 @@
-*.idea
-venv
+*
+
+!*.json
```

Comment:
can we restrict this to the data folder and those file endings? We probably want to change python scripts and so on.

Reply:
right, makes sense